tests/integration/testlib/ovslib.py

```diff
@@ -58,15 +58,18 @@ def add_link_aggregation_port(self, name, slaves, mode=None):
                 OVSBridge.Port.LinkAggregation.MODE
             ] = mode
 
-    def add_internal_port(self, name, ipv4_state):
+    def add_internal_port(self, name, mac=None, ipv4_state=None):
```

Comment:
Please consider adding a `*, ` before `mac` to enforce the keyword arguments and make future changes easier. This change is changing the signature already.

Reply:
Done